test/SmashSpecSM.hs

```diff
@@ -179,7 +179,7 @@ smashSM dataLayer = StateMachine
     mSemantics (InsertPool poolId poolHash poolOfflineMeta) = do
         let addPoolMetadata = dlAddPoolMetadata dataLayer
         -- TODO(KS): Fix this.
-        result <- addPoolMetadata poolId poolHash poolOfflineMeta (PoolTicker "tickerName")
+        result <- addPoolMetadata (panic "smashSM") poolId poolHash poolOfflineMeta (PoolTicker "tickerName")
```

Comment:
I guess we can leave this as-is for now, since the test don't touch the DB for now, just the model.

Reply:
Exactly.

Reply:
Fixed.